src/qae/object-properties.d.ts

```diff
@@ -5,6 +5,7 @@ declare module 'object-properties' {
     version: string;
     qHyperCubeDef: QHyperCubeDef;
     showTitles?: boolean;
+    enableExplorationChart?: boolean;
```

Comment:
should be `enableChartExploration`?

Reply:
Just enableExploration should be fine, chart is redundant here.

Reply:
In sense-client I saw serveral things with "exploration" on it might be difficult to distinguish what exploration means?

Reply:
At the moment there is 2, the old one and the new one, but since we are using exploration in the support property as well as inside client to build the component, i would say we continue with that pattern or rename all to chart-exploration, lets get Veinfors input on this one.

Reply:
`enableChartExploration` or `enableExploration` both sounds OK to me, don't have any strong opinion...

Reply:
Lets use your suggestion then Linh `enableChartExploration` should be fine.

Reply:
```suggestion
    enableChartExploration?: boolean;
```